src/nvim/memory.h

```diff
@@ -2,6 +2,104 @@
 #define NVIM_MEMORY_H
 
 #include <stddef.h>  // for size_t
+#include <string.h>
+#include <strings.h>
```

Comment:
What's this for? I do not think this is available in MSVC, plus some comments suggest it is non-standard.